comps/router/src/requirements.txt

```diff
@@ -0,0 +1,34 @@
+aiofiles
+aiohttp
+docarray[full]
+docx2txt
+# Basic web and async
```

Comment:
Although breaking this file into sections with comments in nice, but we prefer sorted requirements which remove any chances of duplication specially with different versions of same dependency which cause a lot of harm.